### PR TITLE
add XML and PHP config examples

### LINKED ISSUE #13010: [Messenger] missing XML and PHP configuration for failure_transport
**Version** Current

**Description**
Missing documentation for XML and PHP configuration regarding failure_transport
https://symfony.com/doc/current/messenger.html#saving-retrying-failed-messages

**Notes**
XSD does not specify any attribute to set failure_transport
`/vendor/symfony/framework-bundle/Resources/config/schema/symfony-1.0.xsd`

### PR DESCRIPTION
**Pull request by xabbuh**
fixes #13010

### CODE REVIEW

File: messenger.rst [hunk 1]
```diff
@@ -947,13 +1035,47 @@ holds them in memory during the request, which can be useful for testing.
...
+
+        # config/packages/test/messenger.yaml
+        framework:
+            messenger:
+                transports:
+                    async_priority_normal: 'in-memory:///'
+
+    .. code-block:: xml
+
+        <!-- config/packages/messenger.xml -->
```
Comment (javiereguiluz) [inline messenger.rst]: Should this be `config/packages/test/messenger.xml`?

File: messenger.rst [hunk 2]
```diff
@@ -947,13 +1035,47 @@ holds them in memory during the request, which can be useful for testing.
...
+            <framework:config>
+                <framework:messenger>
+                    <framework:transport name="async_priority_normal" dsn="in-memory:///"/>
+                </framework:messenger>
+            </framework:config>
+        </container>
+
+    .. code-block:: php
+
+        // config/packages/messenger.php
```
Comment (javiereguiluz) [inline messenger.rst]: Same for this path.

File: messenger.rst (hunk 1 above)
Comment (xabbuh) [reply messenger.rst]: absolutely, thanks for the heads up 👍

**Review by OskarStark — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Thank you Christian.

**Pull request merged**